core/src/com/unciv/ui/mapeditor/MapEditorMenuPopup.kt

```diff
@@ -11,12 +12,32 @@ import com.unciv.ui.utils.*
 class MapEditorMenuPopup(var mapEditorScreen: MapEditorScreen): Popup(mapEditorScreen) {
 
     init {
-        addButton("New map", 'n') { UncivGame.Current.setScreen(NewMapScreen(mapEditorScreen.tileMap.mapParameters)) }
-        addButton("Save map", 's') { mapEditorScreen.game.setScreen(SaveAndLoadMapScreen(mapEditorScreen.tileMap, true, mapEditorScreen)); this.close() }
-        addButton("Load map", 'l') { mapEditorScreen.game.setScreen(SaveAndLoadMapScreen(mapEditorScreen.tileMap, false, mapEditorScreen)); this.close() }
-        addButton("Exit map editor", 'x') { mapEditorScreen.game.setScreen(MainMenuScreen()); mapEditorScreen.dispose() }
-        addButton("Change ruleset", 'c') { MapEditorRulesetPopup(mapEditorScreen).open(); close() }
-        addCloseButton()
+        add(("{RNG Seed} " + mapEditorScreen.tileMap.mapParameters.seed.toString()).toLabel()).row()
+        addButton("Copy to clipboard") {
```

Comment:
If all addButtons need a .fillX() at the end, then just add that as part of the addButton function

Reply:
I fixed this in #4113. In case it is merged, one can call defaults on the Menu Popup and apply the fillX to it